os/kernel/init/os_bringup.c

```diff
@@ -278,18 +281,6 @@ static inline void os_do_appstart(void)
 	heapinfo_drv_register();
```

Comment:
Could you move this as well before preapp with another commit in this PR?
Because TASH and another services are started in preapp.